src/main/java/core/InputListener.java

```diff
@@ -1,5 +1,14 @@
 package core;
 
+/**
+ * Functional Interface which only has a function to deal with new method.
+ */
 public interface InputListener {
```

Comment:
the input names are obvious so good job there, but the explanation doesnt really add anything to the javadocs